microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -237,12 +237,19 @@ def _check_supported(self, node: Node) -> None:
             ):
                 raise SkippedException("crashkernel=auto doesn't work for the distro.")
 
-    def _get_resource_disk_dump_path(self, node: Node) -> str:
-        if node.shell.exists(
-            PurePosixPath("/var/log/cloud-init.log")
-        ) and node.shell.exists(PurePosixPath("/var/lib/cloud/instance")):
+    def _get_resource_disk_dump_path(self, node: Node, log: Logger,) -> str:
+        # by default, cloudinit will use /mnt as mount point of resource disk
```

Comment:
I think this logic was handled somewhere, can it be reused?

Reply:
Done.